src/pages/UserPage/UserPage.tsx

```diff
@@ -130,13 +130,25 @@ const UserPage: FC = () => {
       name: 'notify_user',
       label: t('label.user_activation_notification'),
       ariaLabel: t('label.user_activation_notification'),
-      rules: {
-        required: true,
-      },
       className: classes.checkboxInputClass,
     },
   ]
 
+  const onArchiveSubmit: SubmitHandler<FormValues> = useCallback(async () => {
```

Comment:
**[blocker]**

Type here seems to be wrong. This is the type for a submit function of a form, however we are not submitting any form with this function. Same issue with naming. the naming suggest we are submitting a form.